js/app.js

```diff
@@ -5,8 +5,37 @@ if ("serviceWorker" in navigator) {
 }
 
 // Listen to messages from service workers.
-navigator.serviceWorker.addEventListener('message', (event) => {
-    if (event.data.msg === "install") {
-        window.location.replace("/index.html");
-    }
-});
\ No newline at end of file
+if (navigator.serviceWorker) {
+    navigator.serviceWorker.addEventListener('message', (event) => {
+        if (event.data.msg === "install") {
+            window.location.replace("/index.html");
+        }
+    });
+}
+
+if (location.protocol != 'https:') {
+ location.href = 'https:' + window.location.href.substring(window.location.protocol.length);
+}
```

Comment:
Tole bom vrgu vn, ker to itak dela nginx, men pa dela probleme v dev environmentu ker nimam certa na localhostu

Reply:
(torej no loger relevant)